packages/discord.js/src/structures/MessageMentions.js

```diff
@@ -184,17 +184,22 @@ class MessageMentions {
    * @returns {boolean}
    */
   has(data, { ignoreDirect = false, ignoreRoles = false, ignoreEveryone = false } = {}) {
-    if (!ignoreEveryone && this.everyone) return true;
-    const { GuildMember } = require('./GuildMember');
-    if (!ignoreRoles && data instanceof GuildMember) {
-      for (const role of this.roles.values()) if (data.roles.cache.has(role.id)) return true;
+    const user = this.client.users.resolve(data),
```

Comment:
I can't use `resolveId`, because if data will be string id I will not be sure that this is user id and method will work incorrect. Or I need to fetch the user, but that will make this method async.